mod/messageboard/start.php

```diff
@@ -114,7 +114,11 @@ function messageboard_add($poster, $owner, $message, $access_id = ACCESS_PUBLIC)
 			$poster->getURL()
 		), $owner->language);
 
-		notify_user($owner->guid, $poster->guid, $subject, $body);
+		$params = [
+			'action' => 'create',
```

Comment:
Something like `messageboard_add` would save someone from a bunch of object sniffing. Is there a goal to use event names?